onnxruntime/python/tools/transformers/benchmark_helper.py

```diff
@@ -31,6 +31,15 @@ def __str__(self):
         return self.value
 
 
+class OptimizerInfo(Enum):
+    NOOPT = 'no_opt'
```

Comment:
To avoid confusion, let's add some comments to clarify that no_opt means using the raw ONNX model, but OnnxRuntime might still apply optimization as long as graph optimization level is not 0 (disable all).